fe/fe-core/src/main/java/org/apache/doris/planner/BrokerScanNode.java

```diff
@@ -319,12 +319,9 @@ protected TScanRangeLocations newLocations(TBrokerScanRangeParams params, Broker
         brokerScanRange.setParams(params);
         if (brokerDesc.getStorageType() == StorageBackend.StorageType.BROKER
                 || brokerDesc.getStorageType() == StorageBackend.StorageType.OFS) {
-            FsBroker broker = null;
-            try {
-                broker = Env.getCurrentEnv().getBrokerMgr()
-                        .getBroker(brokerDesc.getName(), selectedBackend.getHost());
-            } catch (AnalysisException e) {
-                throw new UserException(e.getMessage());
+            FsBroker broker = Env.getCurrentEnv().getBrokerMgr().getAnyAliveBroker();
```

Comment:
Why discard the `selectedBackend` attribute?
It is useful for multi load.

Reply:
oh, my fault, I'm careless